plugins/arDominionB5Plugin/scss/_layout.scss

```diff
@@ -47,3 +47,36 @@
   margin-top: $spacer;
   padding: $spacer;
 }
+
+.atom-slider-for,
+.atom-slider-nav {
+  .slick-track {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  button {
+    &.slick-arrow.slick-prev {
+      left: -30px;
+    }
+
+    &.slick-arrow.slick-next {
+      right: -30px;
+    }
+  }
+}
+
+.atom-slider-nav {
+  img {
+    max-width: 400px;
+    max-height: 300px;
+  }
+}
+
+.atom-slider-for {
+  .slick-slide {
+    display: flex;
+    justify-content: center;
+  }
+}
```

Comment:
I think this is no longer needed, as it's already being affected by lines 54 to 56.

Reply:
You are correct - this is a remnant of having two sliders each with images.